waku/v2/node/peer_manager/peer_manager.nim

```diff
@@ -246,3 +246,10 @@ proc dialPeer*(pm: PeerManager, peerInfo: PeerInfo, proto: string, dialTimeout =
     return none(Connection)
 
   return await pm.dialPeer(peerInfo.peerId, peerInfo.addrs, proto, dialTimeout)
+
+proc disconnect*(pm: PeerManager, peerInfo: PeerInfo) : Future[bool] {.async.} =
+  let peerId =  peerInfo.peerId
+  debug "Removing peer from manager", peerId = peerInfo.peerId, addr = peerInfo.addrs[0]
+  let resp =  pm.peerStore.addressBook.delete(peerId) and pm.peerStore.protoBook.delete(peerId) and pm.peerStore.keyBook.delete(peerId)
```

Comment:
is peerId always in table?

Reply:
Cant guarantee that. I have added a check. Thanks